### PR TITLE
[UI] Home Page

### PR DESCRIPTION
**Pull request by imjalpreet**
- [x] UI added for the home page.
- [x] It is yet not linked to the other modules.
- [ ] I will soon send a PR with all the forms and also the links.

### CODE REVIEW

File: app/views/home.php [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="<?php echo base_url(); ?>static/css/bootstrap.min.css" rel="stylesheet" media="screen">
+    <link href="<?php echo base_url(); ?>static/css/sidebar.css" rel="stylesheet" media="screen">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
+    <script src="<?php echo base_url(); ?>static/js/bootstrap.js"></script>
+    <style type="text/css">
+
+    </style>
```
Comment (nisargjhaveri) [inline app/views/home.php:13]: Not actually a big problem.. But why an empty style tag! :P

**Comment by nisargjhaveri** [comment]
Looks fine..

File: app/views/home.php (hunk 1 above)
Comment (imjalpreet) [inline app/views/home.php:13]: Actually I had written something in that but then I removed it :p But I think I forgot to remove the tags :p

**Comment by gauravmittal1995** [comment]
The UI looks excellent. Nice work @imjalpreet :+1:

**Comment by gauravmittal1995** [comment]
@imjalpreet Hey, just one correction, can you move the missing_person module from the last to the first. Also for the camp. We need to give a preference order to the modules in order of their importance for accessibility. In this case, its Missing Person > Organisations > Camps > Assets > Requests.

Otherwise its more than perfect!!

**Comment by imjalpreet** [comment]
@gauravmittal1995 Thanks! I will update it soon.

**Pull request merged**